src/Symfony/Component/BrowserKit/Cookie.php

```diff
@@ -202,13 +202,13 @@ private static function parseDate(string $dateValue): ?string
         }
 
         foreach (self::DATE_FORMATS as $dateFormat) {
-            if (false !== $date = \DateTime::createFromFormat($dateFormat, $dateValue, new \DateTimeZone('GMT'))) {
+            if (false !== $date = \DateTimeImmutable::createFromFormat($dateFormat, $dateValue, new \DateTimeZone('GMT'))) {
                 return $date->format('U');
             }
         }
 
         // attempt a fallback for unusual formatting
-        if (false !== $date = date_create($dateValue, new \DateTimeZone('GMT'))) {
+        if (false !== $date = date_create_immutable($dateValue, new \DateTimeZone('GMT'))) {
```

Comment:
let's replace all similar calls by `new \DateTimeImmutable()` if that's the same- it is right?

Reply:
Not really, if the $dateValue is not valid date_create_immutable return false, instead if you use directly the constructor an exception is thrown.